index.js

```diff
@@ -334,6 +334,18 @@ function wrapInput (func) {
   }
 }
 
+function wrapDeserialize (func) {
```

Comment:
this replicates a bit of what is in `wrapInput` maybe we could use something like this instead
```
function wrapDeserialize (func) {
  func = wrapInput(func)
  return function () {
    const r = func.apply(null, arguments)
    if (r === 0) {
      throw new Error('Deserialize err')
    }
  }
}
```

what do you think?

Reply:
You are right. I fixed [it](https://github.com/dfinity/js-bls-lib/pull/6/commits/362393ee3f2872872502bf114a164c7a7f702d46).

Reply:
awesome! looking good :+1: